tests/providers/celery/executors/test_celery_executor.py

```diff
@@ -358,3 +358,12 @@ def test_sentinel_kwargs_loaded_from_string():
     assert default_celery.DEFAULT_CELERY_CONFIG["broker_transport_options"]["sentinel_kwargs"] == {
         "service_name": "mymaster"
     }
+
+
+@conf_vars({("celery", "task_acks_late"): "True"})
```

Comment:
True is the default value, you need to test with False, you can also create a parametrized test to test with both values.

Reply:
> True is the default value, you need to test with False, you can also create a parametrized test to test with both values.

Right, sorry. See: [4a847ec](https://github.com/apache/airflow/pull/37066/commits/4a847ecb1863b5035ac5d09e604ff12493a45e5e)